src/main/java/io/github/onecx/workspace/bff/rs/controllers/MenuItemRestController.java

```diff
@@ -0,0 +1,130 @@
+package io.github.onecx.workspace.bff.rs.controllers;
+
+import java.util.List;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import jakarta.validation.ConstraintViolationException;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.core.GenericType;
+import jakarta.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.resteasy.reactive.RestResponse;
+import org.jboss.resteasy.reactive.server.ServerExceptionMapper;
+
+import gen.io.github.onecx.workspace.bff.clients.api.MenuInternalApi;
+import gen.io.github.onecx.workspace.bff.clients.api.ProductInternalApi;
+import gen.io.github.onecx.workspace.bff.clients.api.WorkspaceInternalApi;
+import gen.io.github.onecx.workspace.bff.clients.model.*;
+import gen.io.github.onecx.workspace.bff.rs.internal.MenuItemApiService;
+import gen.io.github.onecx.workspace.bff.rs.internal.model.*;
+import io.github.onecx.workspace.bff.rs.mappers.*;
+
+@ApplicationScoped
```

Comment:
Add `@LogService` annotation.
Example: https://github.com/onecx/onecx-workspace-svc/blob/main/src/main/java/io/github/onecx/workspace/rs/internal/controllers/MenuInternalRestController.java#L29

Create `LogParam`  implementation for all input attributes:
Example: https://github.com/onecx/onecx-workspace-svc/blob/main/src/main/java/io/github/onecx/workspace/rs/internal/log/MenuItemLogParam.java